modules/messages/server/services/message-to-influx.server.service.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+/**
+ * Module dependencies.
+ */
+var path = require('path'),
+    async = require('async'),
+    config = require(path.resolve('./config/config')),
+    mongoose = require('mongoose'),
+    influxService = require(path.resolve('./modules/core/server/services/influx.server.service')),
+    textProcessor = require(path.resolve('./modules/core/server/controllers/text-processor.server.controller'));
+
+// eslint complained here about unused variable => requiring without assignment
+require(path.resolve('./modules/messages/server/models/message.server.model'));
+
+
+var Message = mongoose.model('Message');
```

Comment:
I meant that `Message` but never mind, I see it is being used. Just a bit lost in Github. ;-)